server/utils/nunjucksSetup.ts

```diff
@@ -250,6 +262,27 @@ export function registerNunjucks(app?: express.Express): Environment {
       : undefined
   })
 
+  njkEnv.addGlobal(
+    'lookupCurfewConditionValue',
+    (additionalCondition: AdditionalCondition | AdditionalCondition[], inputName: string): string[] => {
+      if (!additionalCondition) return []
```

Comment:
Would be good to extract this in to a function, maybe in `/handlers/curfew`? and test it in isolation